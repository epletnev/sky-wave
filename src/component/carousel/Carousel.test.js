import React from "react";
import HomePageCarousel from "./Carousel";
import renderer from "react-test-renderer";

test("should render HomePageCarousel component", () => {
  const tree = renderer.create(<HomePageCarousel />).toJSON();
  expect(tree).toMatchSnapshot;
});
