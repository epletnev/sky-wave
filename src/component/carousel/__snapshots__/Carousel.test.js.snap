// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expext to render HomePageCarousel component 1`] = `<HomePageCarousel />`;
